docs/infrastructure/deployment-targets/amazon-ecs-cluster-target.md

```diff
@@ -17,10 +17,48 @@ Refer to the [AWS documentation](https://docs.aws.amazon.com/AmazonECS/latest/de
 3. Enter a display name for the Amazon ECS Cluster.
 4. Select at least one [environment](/docs/infrastructure/environments/index.md) for the target.
 5. Select at least one [target role](/docs/infrastructure/deployment-targets/index.md#target-roles) for the target.
-6. In the **ECS Cluster** section:
+6. In the **Authentication** section (see the [authentication](#authentication) section for more information):
```

Comment:
Might want to avoid the repeat of "authentication section" here, especially since it is referring to two different things?

```suggestion
6. In the **Authentication** section (see [Authentication](#authentication) below for more information):
```